src/views/states/list/components/StateRow.tsx

```diff
@@ -2,37 +2,75 @@ import React, { FC } from 'react';
 import { NodeNetworkStateModelGroupVersionKind } from 'src/console-models';
 
 import { ResourceLink, RowProps, TableData } from '@openshift-console/dynamic-plugin-sdk';
-import { List, ListItem } from '@patternfly/react-core';
+import { List, ListItem, Title } from '@patternfly/react-core';
+import { ExpandableRowContent, Tbody, Td, Tr } from '@patternfly/react-table';
 import { V1beta1NodeNetworkState } from '@types';
+import { useNMStateTranslation } from '@utils/hooks/useNMStateTranslation';
 
-const StateRow: FC<RowProps<V1beta1NodeNetworkState>> = ({ obj, activeColumnIDs }) => {
-  const typesCount = obj?.status?.currentState?.interfaces.reduce((acc, iface) => {
+import InterfacesTable from './InterfacesTable';
+
+const StateRow: FC<RowProps<V1beta1NodeNetworkState, { rowIndex: number }>> = ({
+  obj,
+  activeColumnIDs,
+  rowData: { rowIndex },
+}) => {
+  const [isExpanded, setIsExpanded] = React.useState(false);
+  const { t } = useNMStateTranslation();
+  const interfaces = obj?.status?.currentState?.interfaces;
+
+  const interfacesByType = interfaces?.reduce((acc, iface) => {
     if (acc[iface.type]) {
-      acc[iface.type] += 1;
+      acc[iface.type].push(iface);
     } else {
-      acc[iface.type] = 1;
+      acc[iface.type] = [iface];
     }
```

Comment:
You can replace these lines of the code by just 2 following lines:

```
acc[iface.type] ??= []
acc[iface.type].push(iface)
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment

Reply:
thanks